packages/main/src/plugin/api/webview-info.ts

```diff
@@ -0,0 +1,29 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+export interface WebviewInfo {
+  id: string;
+  uuid: string;
+  viewType: string;
+  sourcePath: string;
+  icon: string | { readonly light: string; readonly dark: string } | undefined;
```

Comment:
Not related to this PR. We are starting using a lot this type for icons and colors, we might want to create a type definition somewhere and reuse it

<img width="490" alt="image" src="https://github.com/containers/podman-desktop/assets/42176370/a1d7befa-7810-4f26-aa84-684cbff0c0c0">

Reply:
yes